packages/datadog-plugin-rhea/src/index.js

```diff
@@ -118,30 +118,27 @@ function wrapDeliveryUpdate (update) {
 }
 
 function patchCircularBuffer (proto, instrumenter) {
-  Object.defineProperty(proto, 'session', {
+  Object.defineProperty(proto, 'outgoing', {
     configurable: true,
     get () {},
-    set (session) {
-      Object.defineProperty(this, 'session', {
+    set (outgoing) {
+      Object.defineProperty(this, 'outgoing', {
```

Comment:
It would be safer to keep the original property in case the property is assigned multiple times.

Reply:
I don't think we care if it's assigned multiple times, since we only need to patch circular buffer constructor once.